whisper/whisperv5/api.go

```diff
@@ -374,17 +374,17 @@ type PostArgs struct {
 	Payload  hexutil.Bytes `json:"payload"`
 	WorkTime uint32        `json:"worktime"`
 	PoW      float64       `json:"pow"`
-	FilterID uint32        `json:"filterID"`
-	PeerID   hexutil.Bytes `json:"peerID"`
+	FilterID string        `json:"filterid"`
```

Comment:
It does matter, because you're defining an API that shouldn't change (especially not silly errors like these). Please fix.